app/views/travel_advice/_first_part.html.erb

```diff
@@ -14,8 +14,15 @@
     <img src="<%= content_item.map["url"] %>" alt="<%= content_item.map["alt_text"] %>" class="map-image">
     <% if content_item.map_download_url %>
       <figcaption>
+        <%
+          download_text = "Download a more detailed map (PDF"
+          if content_item.map_download_file_size
+            download_text += ", #{number_to_human_size(content_item.map_download_file_size)}"
+          end
+          download_text += ")"
```

Comment:
I think we should assume the file size field is always present:

```suggestion
          download_text = "Download a more detailed map (PDF #{content_item.map_download_file_size}"
```

But that does beg the question, is there a corresponding PR for Travel Advice Publisher to make that work?

Reply:
Have add backend change here

https://github.com/alphagov/travel-advice-publisher/pull/2394